<!-- -*- mode: XML -*- -->
<launch>

    <!-- Set to 1 if you want to run the map_server -->
    <arg name="map_server" default="1" />

    <arg name="car_name" default="car" />
	<arg name="racecar_color" default="" />
    <!-- Launch  map server-->
    <group if="$(arg map_server)">
        <include file="$(find mushr_base)/launch/includes/map_server.launch" />
    </group>

    <!-- Launch car -->

    <group ns="$(arg car_name)">
        <remap from="/$(arg car_name)/initialpose" to="/initialpose" />

        <include file="$(find mushr_sim)/launch/single_car.launch">
            <!-- Could be racecar-mit, racecar-uw-tx2, or racecar-uw-nano -->
            <arg name="racecar_version" value="racecar-uw-nano" />
            <!-- The colors of the racecar, should be of the form "-<platform_color>-<inset_color>" -->
            <!-- An empty string will result in the default URDF -->
              <!-- Check CMakeLists.txt of mushr_description for appropriate values -->
            <arg name="racecar_color" value="$(arg racecar_color)" />

            <arg name="car_name" value="$(arg car_name)" />
         </include>
    </group>

</launch>
